packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -212,10 +332,10 @@ export const getDataElementsFromCalculateOperationConfig = config =>
 export const calculateOperationForAnalytics = async (models, analytics, config) => {
   const { operator } = config;
   if (SINGLE_ANALYTIC_OPERATORS.includes(operator)) {
-    return performSingleAnalyticOperation(analytics, config);
+    return performSingleAnalyticOperation(analytics, config, models);
   }
   if (ARITHMETIC_OPERATORS.includes(operator)) {
-    return performArithmeticOperation(analytics, config);
+    return performArithmeticOperation(analytics, config, models);
```

Comment:
One last thing: it's a common convention to pass dependencies first, then data. We also follow this convention with `models` - they are the first argument in most cases.

Let's also follow this convention here! There will be a very small regression risk for functionality in line 341 (`PERATORS[operator](analytics, config, models)`), but should be very easy to test

Reply:
Except in this case not all operators use models so models is an optional argument here?

Reply:
Oh I see. I'd probably still pass the models as the first argument and then just not use it in the operator utils, but up to you - I'm fine with it staying as is